pkg/i18n/english.go

```diff
@@ -771,6 +771,9 @@ func addEnglish(i18nObject *i18n.Bundle) error {
 		}, &i18n.Message{
 			ID:    "stashOptions",
 			Other: "Stash options",
+		}, &i18n.Message{
+			ID:    "notARepository",
+			Other: "Error: must be run inside a repository",
```

Comment:
I think "Error: must be run inside a **git** repository" is a better choice, there are more things that are called a repository.

Reply:
True, updating